### ISSUE TITLE
Setting title is ignored from review

### ISSUE THREAD

**Issue by sethladd** [reenhanced/gitreflow#127]
```
~/Code/dartdoc[sl-lib-warning] $ git reflow review -t "don't print a warning if an anon lib doesn't have docs"

Review your PR:
--------
Title:
sl-lib-warning

Body:

--------
Submit pull request? (Y)
```

Notice that I set `-t` from the command line, but the branch name is still used as a title (and the EDITOR popped up).

I would have expected that `-t` would mean that I wouldn't have seen the EDITOR pop up, and that the Title would match the command line.

Bug?

**Comment by codenamev** [comment]
Not setting the title is a definite bug.  Is it expected behavior to set the body to the title in this case?

**Comment by sethladd** [comment]
Keeping the body blank is fine. I've never needed or wanted to set the body to something via the command line. I can always do that via the GUI.

**Comment by codenamev** [comment]
Fixed in `0.7.2`
